index.ts

```diff
@@ -43,7 +53,7 @@ const reassignEventListeners = (ws, oldWs, listeners) => {
 const ReconnectingWebsocket = function(
     url: string,
     protocols?: string|string[],
-    options: Object = {}
+    options = <Options>{}
```

Comment:
Just curious, is there any difference between `options: Options = {}` and `options = <Options>{}` ?